.github/workflows/ci.yml

```diff
@@ -39,11 +39,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.18.x'
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v2.3.0
+        uses: golangci/golangci-lint-action@v3
```

Comment:
we should make sure the makefile is installing the same golangci linter

Reply:
bumped version in make file